hamilton/plugins/pandas_extensions.py

```diff
@@ -24,7 +24,7 @@
 
 from sqlite3 import Connection
 
-from pandas._typing import NpDtype
+from pandas._typing import FilePath, NpDtype, ReadBuffer, StorageOptions, WriteBuffer
```

Comment:
For python 3.7 compatibility it might be simpler to drop the ones that aren't in pandas + python 3.7?

Reply:
I have removed it. Thought this was a better implementation so I tried it but failed the Python 3.7 check.